Can't get a full docker steps output on Linux

When running `brick build` on my local Linux machine, I can't seem to get the `docker build` steps output that seem to be always present in the [Drone builds](https://drone.tmrow.com/tmrowco/tmrow/3341/1/3).

Here is an example of what I get running with the latest master `brick` build and the `--verbose` flag:

```
fbarl@fbarl-XPS-15-9550 ~/Projects/tmrowco/tmrow/lighthouse.tmrow.com $ brick --verbose build
2020-05-29 15:28:50,288 [DEBUG] Using yarn cache located at /usr/local/share/.cache/yarn/v6
2020-05-29 15:28:50,288 [INFO] 🔨 Preparing lighthouse.tmrow.com..
2020-05-29 15:28:50,293 [DEBUG] docker build . --iidfile /tmp/tmpipjw7itq -f /home/fbarl/Projects/tmrowco/tmrow/.brickdockerfile
2020-05-29 15:28:53,354 [DEBUG] Tagging as lighthouse.tmrow.com_prepare:latest..
2020-05-29 15:28:53,387 [DEBUG] Tagging as lighthouse.tmrow.com_prepare:fbarl-3135-infer-json-schema-from-realm..
2020-05-29 15:28:53,408 [INFO] 💯 Preparation phase done!
2020-05-29 15:28:53,408 [INFO] 🔨 Building lighthouse.tmrow.com..
2020-05-29 15:28:53,452 [DEBUG] docker build . --iidfile /tmp/tmptdmbykri -f /home/fbarl/Projects/tmrowco/tmrow/.brickdockerfile
2020-05-29 15:30:01,694 [DEBUG] rpc error: code = Unknown desc = executor failed running [/bin/sh -c yarn build]: exit code: 1
2020-05-29 15:30:01,699 [ERROR] docker build . --iidfile /tmp/tmptdmbykri -f /home/fbarl/Projects/tmrowco/tmrow/.brickdockerfile
rpc error: code = Unknown desc = executor failed running [/bin/sh -c yarn build]: exit code: 1
```

@madsnedergaard seems to get the full steps output when running on his machine so it might depend on the running environment, here's my Docker env:

```
fbarl@fbarl-XPS-15-9550 ~ $ docker version
Client:
 Version:           18.09.7
 API version:       1.39
 Go version:        go1.10.4
 Git commit:        2d0083d
 Built:             Fri Aug 16 14:19:38 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.09.7
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.4
  Git commit:       2d0083d
  Built:            Thu Aug 15 15:12:41 2019
  OS/Arch:          linux/amd64
  Experimental:     false
```

p.s. I wonder if passing the `digest` stream straight to `stdout` [in here](https://github.com/tmrowco/brick/blob/master/brick/__main__.py#L256) could be a viable solution :stuck_out_tongue:

--- a/README.md
+++ b/README.md
@@ -93,3 +93,7 @@ Commands:
   prune
   test
 ```
+
+## Known Issues
+
+* **`brick build` doesn't provide full Docker steps output** - this issue might appear for some versions of Docker but it shouldn't be there on the version `19.03.4` (see [#35](https://github.com/tmrowco/brick/issues/35) for more context); the ultimate cause might be a bug with [docker-py](https://github.com/docker/docker-py)

